ow_dynamic_terrain/src/DynamicTerrainVisual.h

```diff
@@ -0,0 +1,66 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+#include "DynamicTerrainBase.h"
+#include "TerrainModifier.h"
+#include <gazebo/rendering/RenderingIface.hh>
+
+namespace ow_dynamic_terrain
+{
+class DynamicTerrainVisual : public gazebo::VisualPlugin, public DynamicTerrainBase
+{
+public:
+  DynamicTerrainVisual();
+
+  void Load(gazebo::rendering::VisualPtr /*visual*/, sdf::ElementPtr /*sdf*/) override;
```

Comment:
Minor: why not just omit the parameter names instead of commenting them out?

Reply:
This was existing code, but it got split into .h and .cpp files. I would prefer to do this sort of cleanup in a separate ticket.